### ISSUE TITLE
./train/fetch_caffe.sh gets different version of Caffe

### ISSUE THREAD

**Issue by leekkww** [richzhang/colorization#40]
It seems that `./train/fetch_caffe.sh` doesn't get the modified version of Caffe that it is supposed get. The downloaded tar file opens to to a directory called `caffe_private_pascal/` (instead of `caffe-colorization/`, which is what is called by `./train/train_model.sh`), and the presets for `Makefile.config.example` does not match with what the model asks for. I had to change a bunch of things after Googling really hard and comment/uncomment a bunch of other things to get the model to to work. Did you remove the supported version of Caffe?

**Comment by yuanyang7** [comment]
I got the same problem.😖 How to solve it..

**Comment by chnagi** [comment]
I too got the same problem. Did Anyone of you resolved this issue? Please let me know

**Comment by xuyifangreeneyes** [comment]
I also got the same problem. QAQ

**Comment by alwyz** [comment]
Yep same problem here. Does not work. Training is basically broken on this script until this is fixed...

**Comment by alwyz** [comment]
OK, so....

I successfully downloaded and built caffe-colorization with this method on a new server with python 2.7 pre-installed. I am actually training for the first time!

I have cuda 8.0 installed . Cudnn v4 (tho i dont think cudnn matters, since i disable it in the makefile)

https://chunml.github.io/ChunML.github.io/project/Installing-Caffe-Ubuntu/

Stop following install info on that site when it gets to the part about installing caffe.

Do the following instead

git clone -b master --single-branch https://github.com/richzhang/colorization.git

run ./train/fetch_init_model.sh

Then, modify /train/fetch_caffe.sh file to use this file in wget line:

https://people.eecs.berkeley.edu/~rich.zhang/projects/2016_colorization/files/train/caffe-colorization_old.tar.gz

run ./train/fetch_caffe.sh

then, in caffe-colorization directory:

modify makefile.config with the following

Change CV settings to comment the current one, and uncomment the #3 one several lines below

INCLUDE_DIRS := $(PYTHON_INCLUDE) /usr/local/include /usr/include/hdf5/serial
LIBRARY_DIRS := $(PYTHON_LIB) /usr/local/lib /usr/lib /usr/lib/x86_64-linux-gnu/hdf5/serial/

comment out the matlab line (i dont have matlab so...)

uncomment python layer:=1

then

make all & make test && make runtest && make pycaffe

dont forget to add your path to your new caffe to bashrc

sudo vim ~/.bashrc

export PYTHONPATH=$HOME/colorization/colorization/caffe-colorization/python:$PYTHONPATH
export PYTHONPATH=$HOME/colorization/colorization/resources:$PYTHONPATH

and then run 

source ~/.bashrc

There's still a few post install bugs that are googleable. But this got me going in the right direction. Hope it helps someone else.
